python/lsst/sims/GalSimInterface/galSimDetector.py

```diff
@@ -34,18 +35,35 @@ def __init__(self, afwDetector, afwCamera, obs_metadata, epoch):
 
         @param [in] epoch is the epoch in Julian years of the equinox against
         which RA and Dec are measured
+
+        @param [in] photParams is an instantiation of PhotometricParameters
+        (it will contain information about gain, exposure time, etc.)
         """
```

Comment:
I don't want users to be passing in that kwarg.  It's just there so that when ```_newOrigin``` calls ```__init__```, it doesn't end up re-instantiating a TAN WCS.  I will include a note in the docstring that users should ignore that kwarg.